docker/hub/Dockerfile.eduserver

```diff
@@ -0,0 +1,13 @@
+FROM matrixdotorg/dendrite:latest AS base
+
+FROM alpine:latest
+
+COPY --from=base /build/bin/dendrite-edu-server /usr/bin
+
+WORKDIR /etc/dendrite
+
+ENTRYPOINT ["/usr/bin/dendrite-edu-server", "--config=dendrite.yaml"]
```

Comment:
Unclear where `dendrite.yaml` comes from. We only copied the binary from `base` so uh...?

Reply:
I've added a readme.